app/src/main/java/com/hedvig/app/util/apollo/PaymentMethodsApiResponseAdapter.kt

```diff
@@ -3,13 +3,19 @@ package com.hedvig.app.util.apollo
 import com.adyen.checkout.components.model.PaymentMethodsApiResponse
 import com.apollographql.apollo.api.CustomTypeAdapter
 import com.apollographql.apollo.api.CustomTypeValue
+import com.hedvig.app.util.toJsonObject
 import org.json.JSONObject
 
 class PaymentMethodsApiResponseAdapter : CustomTypeAdapter<PaymentMethodsApiResponse> {
```

Comment:
Maybe consider adding some comment on why this exists in the first place? The whole context on why we couldn't use a simple string and how this solves the problem we were facing? For future reference since otherwise this feels non-obvious why it exists without the context behind its creation.